markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -183,7 +183,7 @@ library PerpsAccount {
             .toInt();
 
         availableMargin = getAvailableMargin(self, stalenessTolerance) - totalLiquidationReward;
-        isEligible = availableMargin < 0;
+        isEligible = availableMargin < 0 && self.debt > 0;
```

Comment:
Hey yeah, the user should be liquidated if his available margin is below zero. However, to lean on the situation, where users simply add 1 wei margins and liquidate themselves to earn the liquidation reward (which exceeds 1 wei) debt is needed. Debt can't be fabricated, you need to have an open position for debt to materialize. If you open a position, you need collateral to back it, and 1 wei won't do... So you will need to abide by the minimum margin requirements (50 bucks). Hopefully this clears it?

Reply:
> If you open a position, you need collateral to back it, and 1 wei won't do... So you will need to abide by the minimum margin requirements (50 bucks).

In the event, then, that the margin requirement is less than the keeper liquidation reward, tests should reflect that liquidating oneself if profitable. 

This may honestly reveal an important system invariant that should **always** be held:
`margin_requirement > keeper_liquidation_reward` 

> assuming `keeper_liquidation_reward` is always denominated and paid out in USD

Reply:
Yeah that would have been ideal, but this will take a bit of time to get implemented. Aside from issues related to the liquidation reward being too high that it block liquidation and locks up dust accounts. I think the other approach is more elegant in terms of always unblocking a liquidation.